docs/docs/resources/variables/cli_env_vars.env

```diff
@@ -0,0 +1,18 @@
+# CLI Environment variables
+#
+# The default setup is shown. These variables are a lower priority
+# alternative to the commands' flags. If a variable is set, the
+# corresponding flag does not have to be specified in commands.
+# Variables marked as required can instead be set as flags.
+#
+# Base directory to the pipelines (default is current working
+# directory)
+KPOPS_PIPELINE_BASE_DIR = .
+# Path to the config.yaml file
+KPOPS_CONFIG_PATH = config.yaml
+# Path to defaults folder
+KPOPS_DEFAULT_PATH = None
```

Comment:
shouldn't `None` be `null`?

Reply:
I am not sure either, but I found this: https://hexdocs.pm/dotenvy/dotenv-file-format.html
you could just try it out and see what happens?

Reply:
https://github.com/theskumar/python-dotenv

Reply:
looks like the concept of _null_ doesn't exist for dotenv, in each case it parses it as a string.

```python
>>> dotenv_values(stream=StringIO("USER=None"))
OrderedDict([('USER', 'None')])
>>> dotenv_values(stream=StringIO("USER=null"))
OrderedDict([('USER', 'null')])
>>> dotenv_values(stream=StringIO("USER="))
OrderedDict([('USER', '')])
```

Reply:
ah, I see you removed the `None` values now 👍🏻